frontend/src/utils/format.ts

```diff
@@ -110,7 +112,16 @@ export function formatShipmentName(
       >,
 ) {
   const month = shipment.labelMonth.toString().padStart(2, '0')
-  return `${shipment.shippingRoute}-${shipment.labelYear}-${month}`
+  return `${shipment.shippingRoute.toString().replace('_TO_', '-')}-${
```

Comment:
For example: `UK_TO_FR  -> UK-FR`

Reply:
suggestion (non-blocking): add the example in a docstring for the function